content/en/docs/pipelines/sdk/parameters.md

```diff
@@ -36,6 +32,8 @@ def my_pipeline(
     my_url: str = 'http://example.com'
 ):
   ...
+  # In the pipeline function body one can use `my_num`, `my_name`, `my_url`
```

Comment:
Suggested revision (speak directly to the user):

In the pipeline function body, you can use the my_num, my_name, and my_url arguments as PipelineParam objects.

Reply:
Thanks. Done